pages/how-to-contribute.md

```diff
@@ -19,16 +20,16 @@ Any questions/ideas don't hesitate to [contact us][contact].
 
 ## Accumulo Repositories
 
-| Repository         | Links                          | Description
-| -------------------| ------------------------------ | -----------
-| [Accumulo][a]      | [Contribute][ac] [Issues][ai]  | Core Project
-| [Website][w]       | [Contribute][wc] [Issues][wi]  | Source for this website
-| [Examples][e]      | [Contribute][ec] [Issues][ei]  | Example code
-| [Testing][t]       | [Contribute][tc] [Issues][ti]  | Test suites such as continuous ingest and random walk
-| [Docker][d]        | [Contribute][dc] [Issues][di]  | Source for Accumulo Docker image
-| [Wikisearch][s]    | [Contribute][sc] [Issues][si]  | Example application that indexes and queries Wikipedia data
-| [Proxy][p]         | [Issues][pi]                   | Apache Thrift service that exposes Accumulo to other languages
-| [Maven plugin][m]  | [Issues][mi]                   | Maven plugin that runs Accumulo
+| Repository         | Links                          | Description |
+| -------------------| ------------------------------ | ----------- |
+| [Accumulo][a]      | [Contribute][ac] [Issues][ai]  | Core Project |
+| [Website][w]       | [Contribute][wc] [Issues][wi]  | Source for this website |
+| [Examples][e]      | [Contribute][ec] [Issues][ei]  | Example code |
+| [Testing][t]       | [Contribute][tc] [Issues][ti]  | Test suites such as continuous ingest and random walk |
+| [Docker][d]        | [Contribute][dc] [Issues][di]  | Source for Accumulo Docker image |
+| [Wikisearch][s]    | [Contribute][sc] [Issues][si]  | Example application that indexes and queries Wikipedia data |
+| [Proxy][p]         | [Issues][pi]                   | Apache Thrift service that exposes Accumulo to other languages |
+| [Maven plugin][m]  | [Issues][mi]                   | Maven plugin that runs Accumulo |
```

Comment:
Since these are optional, I would take it or leave it. I don't think it matters, as long as it's consistent. I think I recently fixed a few lines on the people page that were missing these, but only because all the other lines had them, not because they are needed.

I'm okay if you leave this change in, but it's also fine to leave it unchanged. I don't think it matters.

Reply:
This resolves a warning about possible incompatibility between versions - so it seemed safer to include them, even if optional.

Reply:
Sounds good. What warning, though? Was this in the Jekyll build output, or in a Markdown-aware text editor you use?

Reply:
IDE.  

```
For compatibility reasons all table rows should have borders (pipe symbols) at the start and at the end
```

Reply:
For what it's worth, we specifically use Kramdown for GitHub-flavored-markdown (gfm), which is what Jekyll uses. So we don't really require compatibility with restrictive variants of Markdown that the IDE is trying to warn about. But, if it gets rid of the IDE error, I'm fine with it. I don't ever use an IDE for Markdown (or for the website at all). I'm a little surprised anybody does. I suspect it might be a bit challenging to get it working well in an IDE, given the mix of Ruby, Bundler, Jekyll, HTML, CSS, JavaScript, Markdown, Liquid, YAML, and who knows how many other languages are mixed in that I missed. :smiley_cat:

Reply:
Well, for the spell check alone I find it worth it.  Does not help when I end up with a correctly spelled, but incorrect word but maybe chatGPT will come to my rescue someday.

IntelliJ does provide a preview pain that does give a good rough idea of what the render looks like - some things for our pages don't work, but what does helps getting a rough idea of how things are rendered, tables, font sizes and highlighting which I do find useful.